jobs/build/custom/Jenkinsfile

```
@@ -232,8 +232,9 @@ node {
 
             stage("build images") {
                 if (!any_images_to_build) { return }
+                buildlib.registry_quay_qe_login()
                 command = doozerOpts
-                command += "${include_exclude} --profile ${repo_type} images:build --push-to ${qe_quay_registry}"
+                command += "${include_exclude} --profile ${repo_type} images:build --push-to ${qe_quay_registry} --registry-config=./qe_quay_config"
```

`DOCKER_CONFIG` is a directory (see https://docs.docker.com/engine/reference/commandline/cli/#change-the-docker-directory). `oc --registry-config` is the `config.json` inside that directory (`DOCKER_CONFIG` may have other contents relevant to docker).

```suggestion
                command += "${include_exclude} --profile ${repo_type} images:build --push-to ${qe_quay_registry} --registry-config=./qe_quay_config/config.json"
```